test/Microsoft.ML.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+// TODO: disable test parallelization for this assembly as running test in parallel sometimes cause test host process to crash
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
+
```

Comment:
Can you please add an additional tag after the TODO to help us search for tags associated with this work item. Say something like // TODO: TEST_STABILITY